docs/madr/decisions/009-tracing-policy.md

```diff
@@ -13,6 +13,22 @@ Tracing in Kuma is implemented with Envoy's [tracing
 support](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/observability/tracing).
 At the moment, Kuma supports Zipkin and Datadog.
 
+### Sampling in Envoy
+
+Envoy implements head-based sampling, meaning the originating service (called also `head` or `first`)
```

Comment:
Did you have a chance to make sure it's the same for datadog?

Reply:
I will do this :)

Reply:
Seem that way, I've setup our marketplace demo set front-end sampling to 100 and back-end sampling to 1 and **every** trace contains both front-end and back-end.

![image](https://user-images.githubusercontent.com/2753650/193032212-5003a28b-9e56-48fb-952b-d8bda0f34afd.png)

![image](https://user-images.githubusercontent.com/2753650/193032275-72ebdefa-7e91-4935-96f0-300f95c0d62e.png)

![image](https://user-images.githubusercontent.com/2753650/193032342-a03da989-eda3-41ff-b42b-d907ee166b6e.png)